src/modules/i18n/containers/__tests__/I18nProvider.spec.js

```diff
@@ -27,20 +27,20 @@ describe.skip('I18nProvider', () => {
 
   it('should transform the resources correctly', () => {
     const input = {
-      'module1': {
-        'language1': {
-          'key1': 'lang1-translated1'
+      module1: {
```

Comment:
do we have submodules? Would be nice to test them here as well, if so

Reply:
Don't know but this was only a linting fix in this place. So not part of this task.